.changelog/41a4c0708d4b410392ea999a0a08129a.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "41a4c070-8d4b-4103-92ea-999a0a08129a",
+    "type": "release",
+    "description": "remove deprecated alexaforbusiness from v2",
```

Comment:
```suggestion
    "type": "feature",
    "description": "Remove deprecated `alexaforbusiness` service.",
```

Reply:
(feature is going to be the most applicable here and I believe we used it for previous deprecations)

Reply:
previous iotroborunner used release, anyway I will use feature in this and later remove PRs